src/site/topics/java-vs-python/java-vs-python.rst

```diff
@@ -75,26 +73,27 @@ Variables and Types
 
 .. warning::
 
-    Unlike how in Python you can change the type that the variable has...
+    Unlike how in Python you can change the type that the variable has, you cannot do that in Java; Java will not
+    compile the above code.
 
-    .. code-block:: python
-        :linenos:
+    .. list-table::
+        :header-rows: 1
 
-        # Python --- Reusing variables for different types
-        otherInt = 5
-        otherInt = 'totally Not 5'
+        * - Python
+          - Java
+        * -     .. code-block:: python
+                    :linenos:
 
-    you **cannot** do this in Java.
+                    otherInt = 5
+                    otherInt = 'totally Not 5'
 
-    .. code-block:: java
-        :linenos:
-        :emphasize-lines: 3
+          -     .. code-block:: java
+                    :linenos:
+                    :emphasize-lines: 3
 
-        // Java --- TRYING reusing variables for different types
-        int otherInt = 5;
-        otherInt = "totally Not 5";       // Broken code
+                    int otherInt = 5;
+                    otherInt = "totally Not 5";     // Nope
```

Comment:
maybe _Nope -- compile time error_

Reply:
Legit the only reason it is this short is because it fit into the table nicer. 

Though, I later realized I can ditch line numbers so I will go back to see if things can fit

Reply:
This is perfect. 

One thing I'm going to emphasize a more this year in 102 is the awesomeness of compile time errors. Students sometimes miss the fact that Python's flexibility can be big trouble. Java won't even let you run certain busted code. This is a good thing.